### DISCUSSION TITLE
Unable to open more then one instance

### DISCUSSION THREAD

**Discussion by moxie-coder** [aiko-chan-ai/DiscordBotClient#120] [Q&A] [answered]
So I don’t know if this was a bug or a feature but I was able to open more then one instance to login to another bot, but now it doesn’t seem to be possible anymore with the latest update, I don’t know if this was intentional or whether it was a bug or a feature

**Comment by aiko-chan-ai** [comment]
it has been a feature since version 3.1 (you shouldn't do that because it can cause memory leaks)

**Reply by moxie-coder** [reply]
oh, there hasn’t been any as far as I know? but also there is no way to login to another bot since the add account menu is broken atm

**Reply by moxie-coder** [reply]
aside from clearing the app data which is a bit of a hassle

**Reply by aiko-chan-ai** [reply, marked as answer]
![image](https://github.com/aiko-chan-ai/DiscordBotClient/assets/71698422/ce2bfbd7-5b53-4437-a9f7-1aaf96684e72)
here

**Reply by aiko-chan-ai** [reply]
![image](https://github.com/aiko-chan-ai/DiscordBotClient/assets/71698422/c08c5c5c-cd07-464f-b746-f06625945c8a)
After adding the token, you can use it

**Reply by moxie-coder** [reply]
nice

**Reply by moxie-coder** [reply]
uhm, so little problem, only the default slash commands are showing up

**Reply by aiko-chan-ai** [reply]
![image](https://github.com/aiko-chan-ai/DiscordBotClient/assets/71698422/03ab466b-be8c-4708-9bc7-0fb9c8b6e591)

**Reply by moxie-coder** [reply]
yeah that’s the thing, it’s not showing up
